.github/workflows/hive-tests.yml

```diff
@@ -3,7 +3,8 @@ name: Hive tests
 on: 
   push:
     tags: ['*']
-
+    branches: 
+        - master
```

Comment:
Please change to `branches: [master]`

Reply:
No problem :) 
but in other workflows we use syntax with dash 
https://github.com/NethermindEth/nethermind/blob/master/.github/workflows/build-nethtest-hive-docker-images.yml